docs/ramp_fitting/source/main.rst

```diff
@@ -37,7 +37,11 @@ a given pixel, the final slope is determined as a weighted average from all
 intervals and is written to a file as the primary output product.  In this
 output product, the 4-D GROUPDQ from all integrations is compressed into 2-D,
 which is then merged (using a bitwise OR) with the input 2-D PIXELDQ to create
-the output PIXELDQ.
+the output PIXELDQ.  If the ramp fitting step is run by itself, the output file 
+name will have the suffix '_RampFit'; if the ramp fitting step is run as the 
```

Comment:
:+1: for keeping close to PEP8 - camel case for class names, lower case with underscores for methods and functions. As for achieving consistency we can have a `Model.name` attribute used in the naming of the products. We can implement this in our spare time.